supertokens_python/recipe/session/recipe.py

```diff
@@ -299,21 +299,27 @@ def reset():
             raise_general_exception("calling testing function in non testing env")
         SessionRecipe.__instance = None
 
-    def add_claim_from_other_recipe(self, claim: SessionClaim[Any]):
-        self.claims_added_by_other_recipes.append(claim)
+    @staticmethod
```

Comment:
I think these should be instance methods.

Reply:
Hey @porcellus , I made it static based on https://github.com/supertokens/supertokens-node/pull/352/files#diff-dd812fd78904a37e7eb4dc6156f8e62e23cf966041cfc5d89120378b0dd5e9d6R131

Have we changed that?

Reply:
yeah. I think this was changed in the email verification PR because this way it was more consistent with other things. (and in general I prefer instance methods/members over static)
Check here: https://github.com/supertokens/supertokens-node/blob/session_grants/lib/ts/recipe/session/recipe.ts

Reply:
This was changed in the EmailVerification PR so it'll be implemented at that time (next week)